plugins/LayerView/LayerView.py

```diff
@@ -91,7 +110,7 @@ def beginRendering(self):
 
         if not self._ghost_shader:
             self._ghost_shader = OpenGL.getInstance().createShaderProgram(Resources.getPath(Resources.Shaders, "color.shader"))
-            self._ghost_shader.setUniformValue("u_color", Color(0, 0, 0, 64))
+            self._ghost_shader.setUniformValue("u_color", Color(32, 32, 32, 96))
```

Comment:
We might want to consider finding a way to get these kind of colors out of the hardcoded spectrum (buildplate also has them). For now it's not a really big issue.